assets/schema/ensemble_schema.json

```diff
@@ -193,10 +195,12 @@
             }
           },
           "body": {
-            "type": [
-              "string",
-              "object"
-            ],
+            "type": {
+              "oneOf": [
+                { "type": "string" },
+                { "type": "object" }
```

Comment:
Please add `additionalProperties` of type: string so user can specify custom keys for their body https://rjsf-team.github.io/react-jsonschema-form/docs/usage/objects/#additional-properties. There is an example in the invokeAPI body.

Reply:
done